src/helpers/figures/charts/chart_ui_common.ts

```diff
@@ -213,6 +220,19 @@ export function getChartDatasetFormat(getters: Getters, dataSets: DataSet[]): Fo
   return undefined;
 }
 
+function countStringOccurrences(data) {
```

Comment:
A more natural API for this function would be more something like `{ [string]: [count] }` IMHO (not coupled to the shape of a chart dataset).
It would also be easy to use for charts with `Object.keys()` for labels and `Object.values()` for values (I dare hope the order of `.keys()` and `.values()` is consistent)